docs/tools/developer-tools/lab/api-explorer/horizon-endpoint.mdx

```diff
@@ -0,0 +1,360 @@
+            "href": "https://horizon-testnet.stellar.org/operations/4113586532130817"
+          },
+          "transaction": {
+            "href": "https://horizon-testnet.stellar.org/transactions/bbc6b423efb9c5a8ebb1ac72437a326ffd4b8a202782157f9a9bca38f742f7d7"
+          },
+          "effects": {
+            "href": "https://horizon-testnet.stellar.org/operations/4113586532130817/effects"
+          },
+          "succeeds": {
+            "href": "https://horizon-testnet.stellar.org/effects?order=desc&cursor=4113586532130817"
+          },
+          "precedes": {
+            "href": "https://horizon-testnet.stellar.org/effects?order=asc&cursor=4113586532130817"
+          }
+        },
+        "id": "4113586532130817",
+        "paging_token": "4113586532130817",
+        "transaction_successful": true,
+        "source_account": "GBPIMUEJFYS7RT23QO2ACH2JMKGXLXZI4E5ACBSQMF32RKZ5H3SVNL5F",
+        "type": "payment",
+        "type_i": 1,
+        "created_at": "2024-11-14T20:37:28Z",
+        "transaction_hash": "bbc6b423efb9c5a8ebb1ac72437a326ffd4b8a202782157f9a9bca38f742f7d7",
+        "asset_type": "native",
+        "from": "GBPIMUEJFYS7RT23QO2ACH2JMKGXLXZI4E5ACBSQMF32RKZ5H3SVNL5F",
+        "to": "GDE25LQ34AFCSDMYTOI6AVVEHRXFRJI4MOAVIUGUDUQEC5ZWN5OZDLAZ",
+        "amount": "4.0000000"
+      },
+      {
+        "_links": {
+          "self": {
+            "href": "https://horizon-testnet.stellar.org/operations/4115390418391041"
+          },
+          "transaction": {
+            "href": "https://horizon-testnet.stellar.org/transactions/6de546bd7702f1590f21928e6aa1329582b436f11d9fc9bcce304ea51f1893b0"
+          },
+          "effects": {
+            "href": "https://horizon-testnet.stellar.org/operations/4115390418391041/effects"
+          },
+          "succeeds": {
+            "href": "https://horizon-testnet.stellar.org/effects?order=desc&cursor=4115390418391041"
+          },
+          "precedes": {
+            "href": "https://horizon-testnet.stellar.org/effects?order=asc&cursor=4115390418391041"
+          }
+        },
+        "id": "4115390418391041",
+        "paging_token": "4115390418391041",
+        "transaction_successful": true,
+        "source_account": "GBPIMUEJFYS7RT23QO2ACH2JMKGXLXZI4E5ACBSQMF32RKZ5H3SVNL5F",
+        "type": "payment",
+        "type_i": 1,
+        "created_at": "2024-11-14T21:12:31Z",
+        "transaction_hash": "6de546bd7702f1590f21928e6aa1329582b436f11d9fc9bcce304ea51f1893b0",
+        "asset_type": "native",
+        "from": "GBPIMUEJFYS7RT23QO2ACH2JMKGXLXZI4E5ACBSQMF32RKZ5H3SVNL5F",
+        "to": "GCNJ2TA24PJNDPE2JF3BYVLUTJX2GLRDGGBG4SYMQHKHYDHSMAPRZXWC",
+        "amount": "5.0000000"
+      }
+    ]
+  }
+}
+```
+
+### [All Assets](https://lab.stellar.org/endpoints/assets)
+
+Horizon's `/assets` endpoint provides information about an asset. Since we are not interested in historical data, we will use SDF's mainnet Horizon to retrieve the latest information about the asset.
```

Comment:
Networks were capitalized earlier in the doc.
```suggestion
Horizon's `/assets` endpoint provides information about an asset. Since we are not interested in historical data, we will use SDF's Mainnet Horizon to retrieve the latest information about the asset.
```